openvdb/openvdb/python/README.md

```diff
@@ -0,0 +1,87 @@
+# `openvdb`
+
+This project contains python bindings for OpenVDB.
+
+## Dependencies
+
+| Name    | Version |
+| ---     | ---     |
+| cmake   | >=3.15  |
+| numpy   | >=1.14  |
+| openvdb | >=9.0.0 |
```

Comment:
I find this confusing. Aren't the bindings made specifically for a specific OpenVDB version?

Reply:
This should be removed anyway, but pybind11 support was only merged a little while ago. You won't be able to build this without a recent version. Anyway, this section is removed now.